integration_tests/src/main/python/parquet_test.py

```diff
@@ -125,6 +125,15 @@ def test_ts_read_round_trip(spark_tmp_path, ts_write, ts_rebase, small_file_opt,
             conf={'spark.rapids.sql.format.parquet.smallFiles.enabled': small_file_opt,
                   'spark.sql.sources.useV1SourceList': v1_enabled_list})
 
+def test_parquet_write_ts_millis(spark_tmp_path):
+    gen = TimestampGen()
```

Comment:
What are we doing to avoid `ts_rebaseModeIn*`  We cannot support the full range of write/read options for time stamps unless something has changed recently.

Reply:
We might not be hitting the ambiguous dates while generating the Timegen in this tests as the tests are passing.

Reply:
Upon digging into this further, I am a little confused as to why the tests were passing without passing the ts_rebaseModeIn*. As per the documentation, the default mode is `EXCEPTION` which should throw an exception if it encounters an ambiguous date, but when I don't set a ts_rebaseModeIn* it behaves as if I had set the value as `CORRECTED`. Upon explicitly setting the value to `EXCEPTION` the test threw an exception. I have made the change to explicitly set the ts_rebaseModeIn* as `CORRECTED`. Please let me know if this is acceptable or if you had something else in mind.